hydroshare/settings.py

```diff
@@ -335,6 +325,8 @@
 }
 
 OAUTH2_PROVIDER_APPLICATION_MODEL = 'oauth2_provider.Application'
+OAUTH2_PROVIDER_ACCESS_TOKEN_MODEL = 'oauth2_provider.AccessToken'
```

Comment:
These settings are not relevant to removal of deprecated resource types.  Was this accidentally added?

Reply:
I had to make this change in order for the tests to run without migrations. I have fixed it now as the tests will run with migrations applied.